s4e-web/src/app/state/form/form.model.ts

```diff
@@ -7,6 +7,7 @@ import {SaveConfigForm} from '../../views/map-view/zk/save-config-modal/save-con
 import {ReportForm} from '../../views/map-view/zk/report-modal/report-modal.model';
 import {SentinelSearchForm} from '../../views/map-view/state/sentinel-search/sentinel-search.model';
 import {ShareConfigurationForm} from '../../views/map-view/zk/configuration/state/configuration.model';
+import { ParentInstitutionForm } from 'src/app/views/settings/manage-institutions/parent-institution-modal/parent-institution-modal.model';
```

Comment:
The style is to not have spaces after curly braces eg. `{ParentInstitutionForm}`

Reply:
It's automated by VS to add it this way. As it is inconsistent trough the whole application code, maybe instead repairing it in here, we should use some pre-commit `linter`.